frontend/.husky/pre-push

```diff
@@ -1,5 +1,5 @@
-#!/usr/bin/env sh
-. "$(dirname -- "$0")/_/husky.sh"
-
-cd frontend && npm run coverage
-cd ../backend && ./gradlew clean check
+##!/usr/bin/env sh
```

Comment:
Why delete this file then add it back again
SHOULD NOT comment this git hook

Reply:
done